### PR TITLE
fixes #5809 - don't show parent location/organization/hostgroup select box if zero or one location/organization/hostgroup

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
[test]

File: app/views/taxonomies/_form.html.erb [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 <%= form_for taxonomy do |f| %>
   <%= base_errors_for taxonomy %>
-  <%= select_f f, :parent_id, taxonomy.class.where("id NOT IN (#{taxonomy.subtree_ids.join(',')})").order(:title), :id, :title, { :include_blank => true },
+  <%= select_f(f, :parent_id, taxonomy.class.where("id NOT IN (#{taxonomy.subtree_ids.join(',')})").order(:title), :id, :title, { :include_blank => true },
                { :label => _('Parent'), :onchange => 'parent_taxonomy_changed(this);',
                  :help_inline => :indicator,
                  :'data-url' => (controller_name == 'organizations' ? parent_taxonomy_selected_organization_path(taxonomy.id) :
                                                                       parent_taxonomy_selected_location_path(taxonomy.id))
-               } %>
+               }) if (taxonomy.class.count > 1) %>
```
Comment (ohadlevy) [inline app/views/taxonomies/_form.html.erb]: nitpick: extract to an helper?

**Comment by isratrade** [comment]
@ohadlevy, added helper to hostgroup form, too.

**Comment by ohadlevy** [comment]
:+1: maybe the same helper can be used in the media pr too?

**Comment by domcleal** [comment]
Merged as 92ea9c3 for Foreman 1.6, thanks @isratrade.

**Pull request closed without merging**